src/Log/Utilities/Filesystem.php

```diff
@@ -163,7 +163,7 @@ public function all()
     private function getFiles($pattern)
     {
         $files = $this->filesystem->glob(
-            $this->storagePath . DIRECTORY_SEPARATOR . $pattern, GLOB_BRACE
+            $this->storagePath.DIRECTORY_SEPARATOR.$pattern, GLOB_BRACE
```

Comment:
__[code_sniffer]__
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->